File: TD/TD6/Code/Python/5.py
#!/usr/bin/python
# Calculer la somme des valeurs du tableau


def somme_tab(tab):
    somme = 0

    for i in range(len(tab)):
        somme += tab[i]

    return somme


tab = [5, 18.5, 13.2, 8.75, 2, 15, 13.5, 6, 17]

print(somme_tab(tab))

File: TD/TD6/Code/5.cpp
/* Calculer la somme des valeurs du tableau */
#include <iostream>

/* Procédure */

float sommeTableau(float tab[], int n)
{
  float somme;
  somme = 0;
  
  for (int i = 0; i < n; ++i) {
    somme += tab[i];
  }
  
  return somme;
}

/* Utilisation */

int main()
{
  float tab[9] = {12, 18.5, 13.2, 8.75, 16, 15, 13.5, 12, 17};
  
  std::cout << sommeTableau(tab, 9) << std::endl;
  
  return 0;
}
